test/cmdlineTests/linking_strict_assembly_duplicate_library_name/args

```diff
@@ -0,0 +1 @@
+--strict-assembly --libraries library.sol:L:0x1234567890123456789012345678901234567890,library.sol:L:0x0987654321098765432109876543210987654321
```

Comment:
@chriseth @cameel This makes me wonder why is this in the form of `<lib>:<addr>` and not `<lib>=<addr>`?

Reply:
I don't know but I suspect that maybe the file name was a later addition and just reused the separator? In any case, changing it would make this less ambiguous.